aws/aws-how-to/instances/import-local-vm-to-aws.rst

```diff
@@ -0,0 +1,137 @@
+Importing a local Ubuntu VM to AWS
+==================================
+
+This document will guide you to import local virtual machines to AWS.
+Please refer to the official `AWS Documentation`_ for more details. 
+
+Requirements
+------------
+
+- An AWS account
+- AWS CLI installed and configured
+- A virtual machine exported in Open Virtual Appliance (OVA) format. AWS also supports Virtual Machine Disk (``VMDK``), Virtual Hard Disk (``VHD/VHDX``), and raw.
+
+.. note::
+        Users running Ubuntu on KVM, OpenStack, LXD and other virtualization environments will most likely have ``qcow2`` files and not ``OVA``. Although ``qcow2`` is not supported, it is possible to use ``qemu-img`` to convert it to raw as follows: 
+        
+        ``sudo qemu-img convert -O raw /var/lib/libvirt/images/my-machine.qcow2 my-machine.raw``
+    
+
+
+Upload your OVM machine image to S3
+-----------------------------------
+
+
+Go to S3, create or reuse a bucket and upload the image there. Make a note of your bucket name. 
+
+Create a containers JSON file
+-----------------------------
+
+This file (say ``containers.json``) will specify the bucket and full path to the uploaded image for a single disk machine. In case of a multiple disk machine, each disk has to be defined in this file as a separate container (for instance, separate ``VDMK`` files). If you are importing a raw image, make sure to change the format and the image URL accordingly.
+
+.. code::
+
+    [
+          {
+            "Description": "My imported Ubuntu august 2024",
+            "Format": "ova",
+            "Url": "s3://ubuntu-artifacts/ubuntu22.04.ova"
+          }
+    ]
+
+
+Create an IAM role for the VM import process
+--------------------------------------------
+
+
+Go to IAM, create a role called ``vmimport`` and get the access key. 
+
+
+In the process of creating the IAM role, you will create two files: ``role-policy.json`` and ``trust-policy.json``, as described in the AWS documentation. While ``trust-policy.json`` is used in the role creation process, ``role-policy.json`` will be used to create a policy allowing this role to read objects from the source S3 bucket and upload to the export S3 bucket.
+
+
+
+Import the VM
+-------------
+
+
+.. code::
+
+        aws ec2 import-image --description "My imported ubuntu server VM" --disk-containers "file://containers.json" --profile default --region us-east-1
+
+
+This command will return a task ID (``ImportTaskId``), which can be used to check its status. The task itself will take some time since it will import, convert, launch, and create an ami.
```

Comment:
```suggestion
This command will return a task ID (``ImportTaskId``), which can be used to check its status. The task itself will take some time since it will import, convert, launch, and create an AMI.
```